scoped/post/create-post.md

```diff
@@ -0,0 +1,46 @@
+**Name**
+Create post
+
+----
+
+**Actor** 
+Signed-in user
+
+----
+
+**Subject area**
+Any
+
+----
+
+**Preconditions** 
+User is signed in.
```

Comment:
Will there be any preconditions around user reputation here? Or any other barrier to entry before a user is able to create a post?

Reply:
Users at TL0 or TL1 have limits on the number/frequency of posting, so the frontend will need to ask the backend if this user can post now.  Aside from those considerations, anybody with an account can post.